ui/app/AppLayouts/Wallet/controls/BannerCard.qml

```diff
@@ -0,0 +1,101 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtGraphicalEffects 1.15
+
+import StatusQ.Components 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Control {
+    id: root
+
+    property alias image: image.source
+    property alias title: title.text
+    property alias subTitle: subTitle.text
+    property alias closeEnabled: closeButton.visible
+
+    signal clicked()
+    signal close()
+
+    implicitHeight: 70
+    implicitWidth: 400
+
+    TapHandler {
+        acceptedButtons: Qt.LeftButton
+        enabled: !closeHandler.pressed
+        onTapped: {
+            root.clicked()
+        }
+    }
+    HoverHandler {
+        cursorShape: Qt.PointingHandCursor
+    }
+    background: Rectangle {
+        id: background
+        anchors.fill: parent
+        color: Theme.palette.background
+        radius: 12
+        border.width: 1
+        border.color: Theme.palette.baseColor2
+        layer.enabled: true
+        layer.effect: DropShadow {
+            source: background
+            horizontalOffset: 0
+            verticalOffset: 7
+            radius: 8
+            spread: root.hovered ? 0.3 : 0
+            color: Theme.palette.baseColor2
+        }
+    }
+    contentItem: RowLayout {
+        id: layout
+        spacing: Theme.halfPadding
+        StatusImage {
+            id: image
+            Layout.preferredWidth: 36
+            Layout.preferredHeight: 36
+            Layout.leftMargin: 20
+        }
+        ColumnLayout {
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            Layout.margins: layout.spacing
```

Comment:
The margins here around the individual items are weird; you should set the `root.padding` as follows:

![image](https://github.com/user-attachments/assets/b8dab640-b5f6-423c-86ec-d33f00d4c471)

and then no margins

Reply:
Also `layout.spacing` in your case is `8` while it should be `16`